ops/src/messenger.rs

```diff
@@ -59,29 +60,58 @@ impl Messenger {
         branch: &str,
         commits: &[T],
     ) {
-        self.send_to_channel(msg, attachments, repo, branch, commits);
+        self.send_to_channel(
+            msg,
+            attachments,
+            repo,
+            branch,
+            commits,
+            Vec::<String>::new(),
+            false,
+        );
         self.send_to_slackbots(vec![item_owner.clone()], msg, attachments);
     }
 
+    #[allow(clippy::too_many_arguments)]
     pub fn send_to_channel<T: github::CommitLike>(
         &self,
         msg: &str,
         attachments: &[SlackAttachment],
         repo: &github::Repo,
         branch: &str,
         commits: &[T],
+        thread_urls: Vec<String>,
+        init_thread: bool,
     ) {
+        // We could look at the thread_url to see if threads have been used in the past, but that
```

Comment:
sorry: there's still a bunch of references to `thread_url` -- would be better to update them all to `thread_guid`